entry.rb

```diff
@@ -0,0 +1,19 @@
+require 'colorize'
+
+def languagePrompt
+  puts('Enter'.green + ' NODE'.red + ' or '.green + 'RUBY'.red + '.'.green)
+  input = gets.upcase.sub(/\n/, '')
+
+  if (input === 'NODE')
+    exec 'node', './node/node-menu.js'
```

Comment:
Consider putting this in `package.json` as a script entry, instead of requiring the `ruby` runtime to execute both node and ruby scripts.